shell/platform/glfw/flutter_glfw.cc

```diff
@@ -664,9 +664,8 @@ static bool RunFlutterEngine(
           std::filesystem::path(executable_location) / aot_library_path;
     }
   }
-  std::string assets_path_string = assets_path.u8string();
-  std::string icu_path_string = icu_path.u8string();
-  std::string lib_path_string = aot_library_path.u8string();
+  auto assets_path_string = assets_path.c_str();
+  auto icu_path_string = icu_path.c_str();
```

Comment:
It looks like we can just inline these directly where they are used below instead of having the extra variables. (I think they were left over from a different code structure.)